lib/usdUfe/ufe/UsdHierarchy.cpp

```diff
@@ -189,18 +189,8 @@ Ufe::SceneItemList UsdHierarchy::children() const
 
 Ufe::SceneItemList UsdHierarchy::filteredChildren(const ChildFilter& childFilter) const
 {
-    // Note: for now the only child filter flag we support is "Inactive Prims".
-    //       See UsdHierarchyHandler::childFilter()
-    if ((childFilter.size() == 1) && (childFilter.front().name == "InactivePrims")) {
-        // See uniqueChildName() for explanation of USD filter predicate.
-        const bool             showInactive = childFilter.front().value;
-        Usd_PrimFlagsPredicate flags
-            = showInactive ? UsdPrimIsDefined && !UsdPrimIsAbstract : kUsdUfePrimDefaultPredicate;
-        return createUFEChildList(getUSDFilteredChildren(_item, flags), !showInactive);
-    }
-
-    UFE_LOG("Unknown child filter");
-    return Ufe::SceneItemList();
+    Usd_PrimFlagsPredicate flags = UsdUfe::getUsdPredicate(childFilter);
+    return createUFEChildList(getUSDFilteredChildren(_item, flags), false);
```

Comment:
You are now always passing `false` here when before the code used to have a boolean. How is that correctly filtering the inactive now?

Reply:
They are filtered by the USD predicate. We never needed to pass true here, unlike inthe chidlren() function, which having no USD predicate, need the flag to filter inactive prims.

Reply:
When I edit as Maya, I see the Maya node under the USD node, replacing the USD node, if that is what you meant.

Reply:
I don't recall exactly what that comment was referring to, but thanks for verifying by testing.